TUM-Dev/NavigaTUM
# [Coordinate-picker-rework 4] Addition of a new coordinate picker

Fixes #669

## Proposed Changes (include Screenshots if possible)

- Extracted the Coordinate picking logic to its own component (also prepares us for #501)
- Added an E2E Test to ensure that there will not be regressions.

## How to test this PR

1. Launch the 
2. Submit coordiante feedback
3. See the issue

## How has this been tested?

- [ ] E2E Testing
- [ ] Reproduction

## Checklist
